features/step_definitions/block-consolidation-page-steps.js

```diff
@@ -42,21 +44,28 @@ Then(
     const {
       value: { currentEpoch },
     } = await this.client.executeAsync(done =>
-      done({ currentEpoch: daedalus.stores.networkStatus.currentEpoch })
+      done({ currentEpoch: daedalus.stores.blockConsolidation.currentEpoch })
     );
 
+    let currentEpochText = '';
+    let previousEpochText = '';
+
+    if (action === 'accurately') {
+      currentEpochText = currentEpoch ? ` (${currentEpoch})` : '';
+      previousEpochText = currentEpoch
+        ? ` (${Math.max(currentEpoch - 1, 0)})`
+        : '';
+    }
+
     expect(explanationText).to.equal(
-      `Blocks for the current epoch (${currentEpoch}) and the previous epoch (${Math.max(
-        currentEpoch - 1,
-        0
-      )}) are stored as one file per block. All previous epochs will be consolidated to two files per epoch.`
+      `Blocks for the current epoch${currentEpochText} and the previous epoch${previousEpochText} are stored as one file per block. All previous epochs will be consolidated to two files per epoch.`
```

Comment:
@daniloprates please don't hardcode the content here - we should always match it to the value in the translation file. You can see how we did it here: https://github.com/input-output-hk/daedalus/blob/develop/features/step_definitions/wallets-steps.js#L570